content/blog/2025/logical-options.md

```diff
@@ -0,0 +1,334 @@
+---
+title: How do we move logical shorthands forward?
+sub: There are several proposals, but one major road block
+author: miriam
+date: 2025-03-27
+image:
+  src: blog/2025/closed-sign.jpg
+  alt: >
+    A chain-link gate in black and white
+    with a sign that says
+    closed indefinitely,
+    and a smaller warning
+    with gruesome icons for
+    entrapment (a person being smashed)
+    and pinching (a hand going through gears)
```

Comment:
Would this benefit from a `position` setting? For me the sign is sometimes cut-off at the top.

Reply:
There was no good alignment, so I cropped the image instead. See if that works.